src/Apps/CollectorProfile/Routes/Saves/Components/ArtworkListContent.tsx

```diff
@@ -66,6 +66,14 @@ const ArtworkListContent: FC<ArtworkListContentProps> = ({ me, relay }) => {
     artworks: artworkList?.artworks?.totalCount ?? 0,
   }
 
+  const onChange = useCallback(state => {
+    updateUrl(state, {
+      defaultValues: {
+        sort: defaultSort,
+      },
+    })
+  }, [])
+
```

Comment:
can probably remove this change - was having some infinite looping issues that I traced with profiler to this component but it wasn't only this prop and doesn't explain the issue i was having.